frontend/src/components/ViewActivities/index.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import ActivityList from './activitylist';
 
-/**
- * ViewActivities component.
- */
-const ViewActivities = () => {
-  return (
-    <div>
-      <h1>View Activities</h1>
-    </div>
-  );
-};
+class ViewActivities extends React.Component {
+  /** @inheritdoc */
+  constructor(props) {
+    super(props);
+    this.state = {tripId: props.match.params.tripId};
```

Comment:
I am pretty sure that you don't need state here and could just pass in `props.match.params.tripId` to the tripId prop for the ActivityComponent below.

Reply:
Indeed it does. That's good to know.